repos/system_upgrade/common/libraries/rpms.py

```diff
@@ -114,3 +145,41 @@ def check_file_modification(config):
     """
     output = _read_rpm_modifications(config)
     return _parse_config_modification(output, config)
+
+
+def get_leapp_packages(source_version=None, component=None, include_deps=False):
```

Comment:
As noted in the cabal document, `source_*version` is not good name as:
* it's not clear it is actually major version. just version means X.Y in case of RHEL, which we do not expect.
* There are use-cases when it's needed to get name of package for the different version of system, and than the source_version is confusing as we are interested also about the target major version.

Rename it to `major_version`.

```suggestion
def get_leapp_packages(major_version=None, component=None, include_deps=False):
```
- needs updated rest of the code and docstrings

Reply:
will do